docs/source/adhoc_instances.rst

```diff
@@ -61,3 +61,17 @@ defaults to an ipython repl. Also uses the canary version of the code::
     interactive:
       deploy_group: prod.canary
       mem: 10000
+
+Assuming service role from another AWS account
+----------------------------------------------
+
+If you need to run your instance with another account than your current environment's, e.g. use prod account from dev environment, you can specify ``--cluster`` and ``--assume-role-aws-account``.
+
+In the example below, it's enough to specify ``--cluster`` so ``local-run`` will use configuration that are for ``pnw-prod`` and use prod account, no matter your current environment.
+
+
+.. code-block::sh
+
+    paasta local-run -s <service-name> --pull --assume-pod-identity -i <service-instance> --cluster pnw-prod -I
```

Comment:
optional nit: imo, in documentation, we should try to use the long-form of options so that it's easier for folks to search for in the docs/in paasta --help
```suggestion
    paasta local-run --service <service-name> --pull --assume-pod-identity --instance <service-instance> --cluster pnw-prod --interactive
```